src/zui/ZUITimeline/l10n/messageIds.ts

```diff
@@ -12,6 +12,9 @@ export default makeMessages('zui.timeline', {
     },
   },
   expand: m('show full timeline'),
+  fileUploadErrorMessage: m(
+    "Unable to add note. Possible reason is that it's too long."
```

Comment:
I would rephrase this:

```suggestion
    "Unable to add note. It might be too long."
```

Reply:
Fixed

Reply:
Did you forget to commit this? I can't find it in the PR diff.

Reply:
I did forget, but I've pushed now